ui/apps/platform/src/Components/CVEStackedPill/CVEStackedPill.js

```diff
@@ -84,10 +84,9 @@ const CVEStackedPill = ({
             )}
             {hasScanMessage && (
                 <Tooltip
-                    type="alert"
+                    isContentLeftAligned
                     content={
                         <DetailedTooltipContent
-                            extraClassName="text-alert-800"
```

Comment:
Oh, super! The more rare 800 level classes that we remove, the better for the related accessibility effort.